app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    mask_count = 0
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            mask_count += 1
```

Comment:
The `NotWearingMaskError` is handled correctly by incrementing the mask count, but ensure that the return message is specific and matches the task requirements.